contexts/credentials/v2

```diff
@@ -70,6 +70,11 @@
       "@id": "https://www.w3.org/2018/credentials#digestSRI",
       "@type": "https://www.w3.org/2018/credentials#sriString"
     },
+    "digestMultibase": {
+      "@id": "https://w3id.org/security#digestMultibase",
+      "@type": "https://www.w3.org/2018/credentials#multibase"
```

Comment:
```suggestion
      "@type": "https://w3id.org/security#multibase"
```

Whether we like it or not, at the moment `multibase` is defined in the Security vocabulary. We may have to remedy this and have these two in the same vocabulary at some point, but that is a discussion for another day (ie, post CR).